### PR TITLE
Changed localized string macro to specify bundle

### PR DESCRIPTION
**Pull request by clemke314**
@designatednerd All working on my end. :+1:

### CODE REVIEW

File: FormatterKit/TTTTimeIntervalFormatter.m [hunk 1]
```diff
@@ -120,12 +120,12 @@ - (id)init {
     self.locale = [NSLocale currentLocale];
     self.calendar = [NSCalendar currentCalendar];
 
-    self.pastDeicticExpression = NSLocalizedStringFromTable(@"ago", @"FormatterKit", @"Past Deictic Expression");
```
Comment (designatednerd) [inline FormatterKit/TTTTimeIntervalFormatter.m:123]: Might make sense to just grab a reference to the bundle where this is used repeatedly so you don't have to keep repeating the whole bundle for class bit over and over and over again

Comment (clemke314) [inline FormatterKit/TTTTimeIntervalFormatter.m:123]: Doh!

**Comment by designatednerd** [comment]
One thing you may want to look at for another pull: Are there other similar uses of NSLocalizedString which make this assumption in the library? Might be worth just killing them all off at once in terms of a pull to the main repo.

**Comment by clemke314** [comment]
There's like 60 more using that. Should I do I run through and replace them really quick?

**Comment by designatednerd** [comment]
Yeah, go on ahead - you'll probably need to add a similar bundle for class to each of the files those are in if they have multiple uses.

**Comment by designatednerd** [comment]
Groot to pool

**Pull request merged**